src/main/java/uk/gov/digital/ho/pttg/api/HmrcResource.java

```diff
@@ -64,4 +65,13 @@ private Individual individual(String firstName, String lastName, String nino, Lo
         ninoUtils.validate(sanitisedNino);
         return new Individual(firstName, lastName, sanitisedNino, dob, aliasSurnames);
     }
+
+    private void logRetryCount() {
+        int retryCount = requestHeaderData.retryCount();
+        if (retryCount >= 0) {
+            log.info("Retry count.",
```

Comment:
We don't normally end our log messages with a full stop so maybe remove it for consistency?